### ISSUE TITLE
A problem about learning rate finder

### ISSUE THREAD

**Issue by splash1123** [PyImageSearch/Deep-Learning-for-Computer-Vision-with-Python#6]
_Please use this template while reporting bugs inside DL4CV. Issues related to personal project questions or code that we haven't authored will automatically be closed._

## Are you using one of our development environments? 

We provide a preconfigured AWS AMI and a VM to help our customers skip the environment setup part and jump straight to development. We highly suggest using one of our development environments to run our code. 

If the issue is not related to code, then you can simply skip this question. 

## If you are using your own development environment then please provide the following details.

* GPU model and CUDA version: (skip if not application) 
* OS Platform: 
* Python version: 
* OpenCV version: 
* TensorFlow version:

## Describe the issue 
Hi,
I'm wondering that regarding to this picture (https://pyimagesearch.com/wp-content/uploads/2019/08/learing_rate_finder_lr_plot.png) which is on  August 5, 2019 's post.
When lr=1e-3, the loss is on the way which is decreasing,  on the other hand, when lr = 0.8, the loss is on the way which is increasing. But whether lr = 1e-3 and lr = 0.8 will perform as good as each other since their corresponding losses are the same which is around 1.75. 

Or is there any better explanation for this learning rate finder? 
Because I can not find the answer to this question on dl4cv books, so I'm asking here.

Thanks in advance!

Best,
Drew Yang
